This patch is needed so that build does not fail with errors like:

In file included from ../3rdparty/javascriptcore/JavaScriptCore/wtf/FastAllocBase.h:84:0,
from ../3rdparty/javascriptcore/JavaScriptCore/wtf/Vector.h:24,
from ../3rdparty/javascriptcore/JavaScriptCore/pcre/pcre_exec.cpp:50:
../3rdparty/javascriptcore/JavaScriptCore/wtf/TypeTraits.h:173:69: error: 'std::tr1' has not been declared
template<typename T> struct HasTrivialConstructor : public std::tr1::has_trivial_constructor<T> { };
^

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>

--- qt-everywhere-opensource-src-4.8.6/src/script/script.pro.orig	2016-08-24 00:27:12.079214652 +0000
+++ qt-everywhere-opensource-src-4.8.6/src/script/script.pro	2016-08-24 00:24:42.151217592 +0000
@@ -107,3 +107,6 @@ integrity {
 
 # WebKit doesn't compile in C++0x mode
 *-g++*:QMAKE_CXXFLAGS -= -std=c++0x -std=gnu++0x
+
+# Fixes compilation errors with GCC 6.1.0
+QMAKE_CXXFLAGS += -std=gnu++98
